helm/cloud-provider-vsphere/templates/rbac.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    vsphere-cpi-infra: role-binding
+    {{- include "labels.common" . | nindent 4 }}
+  name: {{ include "resource.default.name" . }}-servicecatalog.k8s.io:apiserver-authentication-reader
+  namespace: {{ include "resource.default.namespace" . }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: extension-apiserver-authentication-reader
```

Comment:
This role is not defined in this chart.

Reply:
This is one of the default kube role: kube-system    extension-apiserver-authentication-reader        2020-10-28T10:11:34Z

Reply:
See https://kubernetes.io/docs/tasks/extend-kubernetes/setup-extension-api-server/

Reply:
I added the comment in the rbac file